update documentation to sync with github wiki (#9516)
opensearch-project/documentation-website: _migration-assistant/migration-phases/backfill.md

From the snapshot you created of your source cluster, you can begin backfilling documents into the target cluster. Once you have started this process, a fleet of workers will spin up to read the snapshot and reindex documents into the target cluster. This fleet of workers can be scaled to increased the speed at which documents are reindexed into the target cluster.

### Cloudwatch Metrics and Dashboard
The Migration Assistant creates a CloudWatch Dashboard you can use to visualize the health and performance of the backfill process.  It brings together the metrics for the backfill workers and, for those migrating to Amazon OpenSearch Service, the target cluster.

> You can find the Backfill dashboard in the AWS Console for CloudWatch Dashboards, in the region you have deployed the Migration Assistant.  The metric graphs for your target cluster will be blank until you select the OpenSearch Domain you're migrating to from the drop-down menu at the top of the Dashboard.

Reviewer: Can we put in the name of the dashboard?

Don't we also have a capture & replay dashboard?

Reviewer: (ah, I just realized that this is because this page is about backfill 👍 )

We recommend slowly scaling up the fleet while monitoring the health metrics of the target cluster to avoid over-saturating it. [Amazon OpenSearch Service domains](https://docs.aws.amazon.com/opensearch-service/latest/developerguide/monitoring.html) provide a number of metrics and logs that can provide this insight.

### Pausing the migration

To pause a migration, use the following command:

```shell
console backfill pause
```

> This stop all existing workers from running while leaving the backfill operation in a state it can be restarted. When it's time to restart the migration, perform one of the following:

Reviewer 2: ```suggestion
This will stop all existing workers from running while leaving the backfill operation in a state from which it can be restarted. When you want to restart the migration, perform one of the following actions:
```

- Run `console backfill start`.
- Scale up the worker count by running `console backfill scale <worker_count>`.

### Stopping the migration

> Completing the backfill process requires manually stopping the migration. Stopping the migration shuts down all workers and cleans up all metadata used to track and coordinate the migration. Once the status checks report your data has been completely migrated, you can stop the migration with the following command:

Reviewer 2: ```suggestion
Completing the backfill process requires manually stopping the migration. Stopping the migration shuts down all workers and cleans up all metadata used to track and coordinate the migration. Once the status checks report that your data has been completely migrated, you can stop the migration with the following command:
```

```shell
console backfill stop
```
{% include copy.html %}

Migration Assistant should return the following response:

```shell
Backfill stopped successfully.
Service migration-aws-integ-reindex-from-snapshot set to 0 desired count. Currently 0 running and 5 pending.
Archiving the working state of the backfill operation...
RFS Workers are still running, waiting for them to complete...
Backfill working state archived to: /shared-logs-output/migration-console-default/backfill_working_state/working_state_backup_20241115174822.json
```

> You cannot restart a stopped migration. Instead you can pause the backfill process using `console backfill pause`.

Reviewer 2: ```suggestion
You cannot restart a stopped migration. Instead, you can pause the backfill process using `console backfill pause`.
```

We recommend slowly scaling up the fleet while monitoring the health metrics of the target cluster to avoid over-saturating it. [Amazon OpenSearch Service domains](https://docs.aws.amazon.com/opensearch-service/latest/developerguide/monitoring.html) provide a number of metrics and logs that can provide this insight.

### Pausing the migration

> If you need to pause the migration, you do so using the command:

Reviewer 3: ```suggestion
To pause a migration, use the following command:
```

Author: looks good

```shell
console backfill pause
```

> This will spin down any and all existing workers while leaving the backfill operation in a state it can be restarted.  When it's time to restart the migration, you can run `console backfill start` or just scale up the worker count to your desired number with `console backfill scale <worker_count>`.

Reviewer 3: ```suggestion
This stop all existing workers from running while leaving the backfill operation in a state it can be restarted. When it's time to restart the migration, perform one of the following:

- Run `console backfill start`.
- Scale up the worker count by running `console backfill scale <worker_count>`.
```

Author: Avoid 'stop' since pausing and resuming conveys different from starting and stopping.

### Stopping the migration

> Completing the backfill process requires manually stopping the migration. Stopping the migration spins down any and all workers and cleans up all metadata used to track and coordinate the migration. Once the status checks report your data has been completely migrated, you can stop the migration with the command:

Reviewer 3: ```suggestion
Completing the backfill process requires manually stopping the migration. Stopping the migration shuts down all workers and cleans up all metadata used to track and coordinate the migration. Once the status checks report your data has been completely migrated, you can stop the migration with the following command:
```

Author: looks good

```shell
console backfill stop
```

Example stop output:

```shell
Backfill stopped successfully.
Service migration-aws-integ-reindex-from-snapshot set to 0 desired count. Currently 0 running and 5 pending.
Archiving the working state of the backfill operation...
RFS Workers are still running, waiting for them to complete...
Backfill working state archived to: /shared-logs-output/migration-console-default/backfill_working_state/working_state_backup_20241115174822.json
```

> You cannot restart a stopped migration.  If you want to pause a migration, use `console backfill pause` instead.

Reviewer 3: ```suggestion
You cannot restart a stopped migration. Instead you can pause the backfill process using `console backfill pause`.
```

Author: looks good

### CloudWatch Metrics and dashboard

> Migration Assistant creates an Amazon CloudWatch dashboard 'MigrationAssistant_ReindexFromSnapshot_Dashboard' that you can use to visualize the health and performance of the backfill process. It combines the metrics for the backfill workers and, for those migrating to Amazon OpenSearch Service, the target cluster.

Reviewer 3: ```suggestion
Migration Assistant creates an Amazon CloudWatch dashboard, named `MigrationAssistant_ReindexFromSnapshot_Dashboard`, that you can use to visualize the health and performance of the backfill process. It combines the metrics for the backfill workers and, for those migrating to Amazon OpenSearch Service, the target cluster.
```

- Run `console backfill start`.
- Scale up the worker count by running `console backfill scale <worker_count>`.

### Stopping the migration

Completing the backfill process requires manually stopping the migration. Stopping the migration shuts down all workers and cleans up all metadata used to track and coordinate the migration. Once the status checks report your data has been completely migrated, you can stop the migration with the following command:

```shell
console backfill stop
```

Example stop output:

```shell
Backfill stopped successfully.
Service migration-aws-integ-reindex-from-snapshot set to 0 desired count. Currently 0 running and 5 pending.
Archiving the working state of the backfill operation...
RFS Workers are still running, waiting for them to complete...
Backfill working state archived to: /shared-logs-output/migration-console-default/backfill_working_state/working_state_backup_20241115174822.json
```

You cannot restart a stopped migration. Instead you can pause the backfill process using `console backfill pause`.

> ### CloudWatch metrics and dashboard

Reviewer 3: ```suggestion
### Amazon CloudWatch metrics and dashboard
```

Author: looks good

We recommend slowly scaling up the fleet while monitoring the health metrics of the target cluster to avoid over-saturating it. [Amazon OpenSearch Service domains](https://docs.aws.amazon.com/opensearch-service/latest/developerguide/monitoring.html) provide a number of metrics and logs that can provide this insight.

### Pausing the migration

If you need to pause the migration, you do so using the command:

```shell
console backfill pause
```

This will spin down any and all existing workers while leaving the backfill operation in a state it can be restarted.  When it's time to restart the migration, you can run `console backfill start` or just scale up the worker count to your desired number with `console backfill scale <worker_count>`.

### Stopping the migration

> Backfill requires manually stopping the migration. Stopping the migration spins down any and all workers and cleans up all metadata used to track and coordinate the migration. Once the status checks report your data has been complete migrated, you can stop the migration with the command:

Reviewer: The first sentence is a little confusing to me.  Maybe clearer as `Completing the backfill process requires manually stopping the migration`?

Additionally `complete migrated` -> `completely migrated`

Author: Agreed, changed in the next commit.

```shell
console backfill stop
```

> Example stop output

Reviewer: nit: colon at the end of this line?

Author: agreed, changed in next commit

```shell
Backfill stopped successfully.
Service migration-aws-integ-reindex-from-snapshot set to 0 desired count. Currently 0 running and 5 pending.
Archiving the working state of the backfill operation...
RFS Workers are still running, waiting for them to complete...
Backfill working state archived to: /shared-logs-output/migration-console-default/backfill_working_state/working_state_backup_20241115174822.json
```

> </details>

Reviewer: Maybe I'm missing it, but do we need an opening `<details>`?

You cannot restart a stopped migration.  If you want to pause a migration, use `console backfill pause` instead.

### Amazon CloudWatch metrics and dashboard

> Migration Assistant creates an Amazon CloudWatch dashboard 'MigrationAssistant_ReindexFromSnapshot_Dashboard' that you can use to visualize the health and performance of the backfill process. It combines the metrics for the backfill workers and, for those migrating to Amazon OpenSearch Service, the target cluster.

Reviewer: Is there a reason this section is more-or-less repeated? Can we combine it with the one on line 85?

Author: Duplicated removed in next commit. Thanks for pointing it out
